### ISSUE TITLE
Could this be extended to a week?

### ISSUE THREAD

**Issue by edward-peters** [frankolson/obsidian-tomorrows-daily-note#3]
Thank you for this useful plugin.
I have been looking for a way to create many future daily notes in one go. Could your plug in be extended so that one could, for example, create a week’s worth of daily notes in one go?

**Comment by frankolson** [comment]
That’s a cool idea I hadn’t thought of! I don’t see why not. I’ll play around with that idea and see what I can do. Thanks!

**Comment by edward-peters** [comment]
Thanks a lot! Roam Research had a plug in for doing this which worked well. If you can make something like this possible it will be a big deal for many of us who want to create future daily notes for a week or more ahead. And save a lot of time!

**Comment by cweekly** [comment]
It's a cool idea. FWIW, I think end users could get most of the way there using Templater...

**Comment by brimwats** [comment]
Yes, I would really appreciate having this plugin extend to next week so I can plan sunday for the work week ahead

**Comment by Gewerd-Strauss** [comment]
+1 for this; best case I am searching for something that keeps generating this weekday's entry for the following week _automatically_ (f.e. on bootup) if not existing anymore - I generate daily's 1-2 weeks in advance for planning purposes.

**Comment by frankolson** [comment]
Hey all, found some time to work on this feature in #21 and the changes were released in version in [1.4.0](https://github.com/frankolson/obsidian-tomorrows-daily-note/releases/tag/1.4.0). Thank you all for your input, I hope this helps out!
